workspaces/vscode-python/Dockerfile

```diff
@@ -35,9 +36,7 @@ RUN apt-get update && \
     gosu curl git htop less nano unzip vim wget zip && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    find /tmp -not -path /tmp -delete && \
-    # Test the gosu installation:
-    gosu nobody true
```

Comment:
We ultimately removed this check from #9831; I removed it from here to stay consistent.